### PR TITLE
Replace the research page with google scholar link.

### LINKED ISSUE #104: Remove research page
I think it's easier and more thorough to find all the papers about ODK through [traditional sources](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=%22open+data+kit%22+or+%22opendatakit%22+or+%22ODK%22&btnG=) so I'd propose removing the research page.

### PR DESCRIPTION
**Pull request by huangyz0918**
Closes #104 

#### What is included in this PR?
Replace the research page with google scholar link.

#### What new issues will need to be opened because of this PR?
No.

#### What is left to be done in the addressed issue?
No.

#### What problems did you encounter?
No.

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -20,6 +20,6 @@ community:
     - title: "History"
       url: /community/history/
     - title: "Research"
-      url: /community/research/
+      url: https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=%22open+data+kit%22+or+%22opendatakit%22+or+%22ODK%22&btnG=
```
Comment (yanokwa) [inline _data/navigation.yml]: or needs to be uppercase before the query actually ORs. Once you do that, you'll see that adding "ODK" adds too much noise, so drop that term. 

It should end up being: `"opendatakit" OR "Open Data Kit"`

Comment (yanokwa) [reply _data/navigation.yml]: But wait until we finish the discussion on the issue before making edits.

Comment (huangyz0918) [reply _data/navigation.yml]: > But wait until we finish the discussion on the issue before making edits.

 Sure.

**Comment by yanokwa** [comment]
Closing because it's no longer relevant.

**Pull request closed without merging**